arangod/RocksDBEngine/RocksDBTransactionState.cpp

```diff
@@ -597,6 +598,14 @@ Result RocksDBTransactionState::triggerIntermediateCommit(bool& hasPerformedInte
   }
 
   hasPerformedIntermediateCommit = true;
+  LOG_DEVEL << "INTERMEDIATE COMMIT "
+            << "insert: " << _numInserts
+            << "update:" << _numUpdates
+            << "removes: " << _numRemoves;
+
+
+
+  ServerStatistics::statistics()._intermediateCommits.fetch_add(_numInserts + _numUpdates + _numRemoves );
```

Comment:
these are definitely *not* the intermediate commits, these are number of document operations.